src/client/redux/reducers/placeholder.js

```diff
@@ -0,0 +1,11 @@
+import { PLACEHOLDER } from '../MSGS';
+
+const initialState = [ ];
```

Comment:
I think `initialState` is better with `{}` object. what do you think !

Reply:
For the whole state, indeed. See src/client/Root.js :)

As to this particular reducer, we will be removing it later. We'll replace it with authReducer. And the initial state there wil be { } as well.

Reply:
oh...I missed that actually !..okay great